lessons/git/install/index.md

```diff
@@ -121,4 +121,16 @@ Dále si můžeš nastavit barevné výpisy – pokud si tedy
 $ git config --global color.ui true
 ```
 
+> [note]
+> Stejně jako ostatní unixové příkazy, ani spuštění `git config` nevypíše žádnou hlášku,
+> že se operace povedla.
```

Comment:
Git se dá pouštět i jinde než na Unixu, a nikde není psáno co to vlastně jsou unixové příkazy. (A zrovna `git` je často dost "ne-Unixově" ukecaný…)

Co takhle?
```
> Spuštění `git config` nevypíše žádnou hlášku, že se operace povedla.
> To je normální; stejně se chová spousta dalších příkazů, např. `cd`.
```